src/components/ListItem.jsx

```diff
@@ -1,5 +1,30 @@
+import { updateItem } from '../api';
+import { subtractDates } from '../utils';
 import './ListItem.css';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+export function ListItem({ name, listPath, id, dateLastPurchased }) {
+	const todaysDate = new Date();
```

Comment:
Consider possibly using the `Timestamp` class instead here

```javascript

import {
	Timestamp,
} from 'firebase/firestore';

const todaysDate = Timestamp.now();

```

See comment below for more details.